sdk/cosmosdb/cosmos/package.json

```diff
@@ -108,17 +108,10 @@
     "@types/tunnel": "^0.0.1",
     "@types/underscore": "^1.8.8",
     "@types/uuid": "^8.0.0",
-    "@typescript-eslint/eslint-plugin": "^2.0.0",
-    "@typescript-eslint/eslint-plugin-tslint": "^2.8.0",
```

Comment:
this should not have been deleted. I am adding the new version: `"@typescript-eslint/eslint-plugin-tslint": "^4.8.1"` here: https://github.com/Azure/azure-sdk-for-js/pull/12581